helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -88,6 +92,19 @@ spec:
           value: {{- if or .Values.django.ingress.activateTLS .Values.django.nginx.tls.enabled }} "True" {{- else }} "False" {{- end }}
         - name: DD_CSRF_COOKIE_SECURE
           value: {{- if or .Values.django.ingress.activateTLS .Values.django.nginx.tls.enabled }} "True" {{- else }} "False" {{- end }}
+        livenessProbe:
+          httpGet:
+            path: /login?next=/
+            port: http
+            {{- if .Values.django.nginx.tls.enabled }}
+            scheme: HTTPS
+            {{- end }}
+            httpHeaders:
+            - name: Host
+              value: {{ .Values.host }}
+          initialDelaySeconds: 120
```

Comment:
I found that 120 is a bit low in some contexts (db needs to be ready too, with migrations etc); could you increase and/or better make it configurable in values.yml? i have recently switched to 600 which is a big much and could be set finer probably. 

 to be as fine as possible, we could set up a startup probe too (see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes )

Reply:
on second thoughts, not sure about that startup probe: defectdojo doesn't really like getting requested before the application is up and running: that tends to generate nasty errors and stack traces. probably better to just use a big enough initialDelay

Reply:
startupProbe is alpha since 1.16 and beta in 1.18. For compatibility, it's better not to use it. Maybe, in the future, add some condition in the helm chart.

Reply:
I'll make initialDelaySeconds configurable.

Reply:
It's now possible to configure livenessProbe of uswgi (which configure readiness on nginx container as well). Is that ok the way I did it?

Reply:
it's great that you added the timeout configuration too and that you increased the default 1s which was a bit low I think! 

for the configuration, I'd have configured all three probes with different values so that looking at the values.yml one immediately understands that there are 3 probes and how to configure them. however if you prefer like this I can live with it :)

Reply:
you could also put the default values directly in the values.yml and no default in the django-deployment.yaml because now it seems there are 2 default values (one of which is commented) but sometimes they differ (like uwsgi.livenessProbe.timeoutSeconds)

Reply:
> for the configuration, I'd have configured all three probes with different values so that looking at the values.yml one immediately understands that there are 3 probes and how to configure them. however if you prefer like this I can live with it :)

Well, nginx is very fast to start and current values should enough even on raspberry! So I don't see the point to configure it. And since readiness on nginx and liveness on uwsgi are probing the same thing, I thought it would be simpler to have only one configuration for both.

Reply:
ok fine!